core/ts/recipe/Graphinator.ts

```diff
@@ -170,7 +170,7 @@ export class Graphinator {
   findRemovedParticles(particles) {
     const runningGraphParticleIds = keys(this.arc.hosts).filter(id => {
       const container = this.arc.hosts[id].meta.container;
-      if (container === 'main#graph') {
+      if (container === 'designer#graph') {
```

Comment:
note: there is no 'universal root' right now, so this is a corner we are cutting (also other places)

Reply:
yes :( acknowledged